conformance_pack/rds.sp

```diff
@@ -232,6 +242,16 @@ control "rds_db_instance_automatic_minor_version_upgrade_enabled" {
   })
 }
 
+control "rds_db_cluster_deletion_protection_enabled" {
+  title       = "RDS clusters should have deletion protection enabled"
+  description = "This control checks whether RDS clusters have deletion protection enabled. This control is intended for RDS DB instances. However, it can also generate findings for Aurora DB instances, Neptune DB instances, and Amazon DocumentDB clusters. If these findings are not useful,then you can suppress them."
+  severity    = "low"
+  sql         = query.rds_db_cluster_deletion_protection_enabled.sql
+
+  tags = merge(local.conformance_pack_rds_common_tags, {
+    other_checks = "true"
```

Comment:
```suggestion
    nist_sp_800_171_rev_2 = "true"
```

I think `other_checks = "true"` is incorrect here.

Reply:
@misraved this is only for other_checks, not applicable for nist one.